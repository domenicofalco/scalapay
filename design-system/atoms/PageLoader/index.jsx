import styled from "styled-components";

export default styled.div`
  position: fixed;
  width: 100%;
  height: 100%;
  left: 0;
  top: 0;
  z-index: 1000;
  background: rgba(255, 255, 255, 0.5);
`;
