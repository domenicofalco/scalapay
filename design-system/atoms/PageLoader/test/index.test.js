import React from "react";
import { cleanup, render } from "@testing-library/react";
import PageLoader from "../";

afterEach(cleanup);

describe("atoms => PageLoader", () => {
  it("It should match snapshot", () => {
    const { asFragment } = render(<PageLoader>PageLoader</PageLoader>);
    expect(asFragment()).toMatchSnapshot();
  });
});
