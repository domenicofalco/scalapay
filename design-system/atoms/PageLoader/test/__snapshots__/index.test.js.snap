// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`atoms => PageLoader It should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="PageLoader-sc-2ikry1-0 gpgDXc"
  >
    PageLoader
  </div>
</DocumentFragment>
`;
